pkg/agent/rest.go

```diff
@@ -115,6 +119,26 @@ func FindRendezvouIPAndSSHKeyFromAssetStore(assetDir string) (string, string, er
 	return rendezvousIP, sshKey, nil
 }
 
+// FindAuthTokenFromAssetStore returns the auth token.
+func FindAuthTokenFromAssetStore(assetDir string) (string, error) {
+	authConfigAsset := &gencrypto.AuthConfig{}
+
+	assetStore, err := assetstore.NewStore(assetDir)
+	if err != nil {
+		return "", errors.Wrap(err, "failed to create asset store")
+	}
+
+	authConfig, authConfigError := assetStore.Load(authConfigAsset)
+
+	if authConfigError != nil {
+		logrus.Debug(errors.Wrapf(authConfigError, "failed to load %s", authConfigAsset.Name()))
```

Comment:
For consistency, it could be useful to just return the error and let the method invoker to trace it. Not sure also if it's worth having it at Info level

Reply:
Updated to return the error. Other assets are also logged at debug level so kept the same. Let me know if it needs to change to Info.